frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -21,12 +21,15 @@ interface IMainLayoutProps {
 const MainLayoutContainer = styled(Grid)(() => ({
     height: '100%',
     justifyContent: 'space-between',
+    display: 'flex',
+    flexDirection: 'column',
+    flexGrow: 1,
+    position: 'relative',
 }));
 
 const MainLayoutContentWrapper = styled('main')(({ theme }) => ({
     margin: theme.spacing(0, 'auto'),
-    overflow: 'auto', // prevent margin collapsing
```

Comment:
I'm worried that this is important, but without removing it `sticky` isn't working